WordPress/Classes/ViewRelated/People/InvitePersonViewController.swift

```diff
@@ -363,6 +368,14 @@ private extension InvitePersonViewController {
         // By default, Send is disabled
         navigationItem.rightBarButtonItem?.enabled = false
     }
+
+    func setupDefaultRole() {
```

Comment:
Problem is: you need the `blog` to be already set. There is no real warranty that it will be available, whenever the property initializer runs.

(That's the reasoning behind this method!)